src/gvizdoom_tests/TestActions.cpp

```diff
@@ -43,13 +43,7 @@ TEST(TestActions, AllActions)
     ASSERT_EQ(doomGame.getGameState<GameState::DamageCount>(), 0);
     ASSERT_EQ(doomGame.getGameState<GameState::BonusCount>(), 0);
     ASSERT_EQ(doomGame.getGameState<GameState::OnGround>(), true);
-    ASSERT_GE(doomGame.getGameState<GameState::X>(), -943.999);
-    ASSERT_LE(doomGame.getGameState<GameState::X>(), -943.0);
-    ASSERT_GE(doomGame.getGameState<GameState::Y>(), 1600.0);
-    ASSERT_LE(doomGame.getGameState<GameState::Y>(), 1600.0934);
-    ASSERT_GE(doomGame.getGameState<GameState::Z>(), -0.001);
-    ASSERT_LE(doomGame.getGameState<GameState::Z>(), 0.001);
```

Comment:
I don't think these should be removed, just replace `doomGame.getGameState<GameState::X>()` with `doomGame.getGameState<GameState::PlayerPos>()(0)` and so on